mtz-update-password.html

```diff
@@ -1,9 +1,11 @@
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="../paper-progress/paper-progress.html">
 <link rel="import" href="../iron-icon/iron-icon.html">
 <link rel="import" href="../iron-validatable-behavior/iron-validatable-behavior.html">
 <link rel="import" href="mtz-validation-status.html">
 <link rel="import" href="mtz-password-validator.html">
+<link rel="import" href="zxcvbn-import.html">
```

Comment:
@stramel is there a way we can easily make this a lazy / conditional import based on opt-in status? This one and progress are only needed if you opt into one feature that shouldn't be required by default

Reply:
That would be nice.